Makefile

```diff
@@ -63,7 +63,8 @@ release-major release-minor release-patch:
 
 .PHONY: setup
 setup:
-	$(NPM) install
+	$(YARN)
+	$(YARN) check
```

Comment:
Oh, I see. For some reason I assumed there to be separate commands for generating the lockfile and installing dependencies, but it makes sense that they happen together.

I'd like to continue to be able to run `make setup` to install dependencies. I suggest this structure:

```make
.PHONY: setup
setup: yarn.lock

yarn.lock: package.json
	$(YARN)
```